sling/core/pckgmgr/src/main/java/com/composum/sling/core/pckgmgr/util/PackageUtil.java

```diff
@@ -686,6 +687,9 @@ public static void toJson(JsonWriter writer, JcrPackageDefinition definition)
         if (lastModified != null) {
             writer.name(JcrPackageDefinition.PN_LASTMODIFIED).value(dateFormat.format(lastModified.getTime()));
         }
+        if (lastUnpacked != null) {
+            writer.name(JcrPackageDefinition.PN_LAST_UNPACKED).value(dateFormat.format(lastUnpacked.getTime()));
```

Comment:
that information in JSON could be consumed by external tooling to determine if package is already installed on instance